depthai_ros_driver/src/conversion.cpp

```diff
@@ -0,0 +1,139 @@
+#include "depthai_ros_driver/conversion.hpp"
+
+namespace rr {
+cv::Mat planar2interleaved(const cv::Mat& mat) {
+    cv::Mat new_mat(mat.size(), mat.type());
+    
+    assert (mat.channels() == 3);
+    const auto WH = mat.cols * mat.rows;
+    
+    const uchar* src = mat.data;
+    uchar* dst = new_mat.data;
+    for (int i = 0; i < WH; i++) {
```

Comment:
```suggestion
    for (int i = 0; i < WH; ++i) {
      dst[i * 3 + 0] = src[i + WH * 0]; // b
      dst[i * 3 + 1] = src[i + WH * 1]; // g
      dst[i * 3 + 2] = src[i + WH * 2]; // r
    }
```

Reply:
thx, updated

Reply:
I cannot see the changes in the latest commit.